akka-actor-typed-tests/src/test/scala/akka/actor/typed/pubsub/LocalPubSubSpec.scala

```diff
@@ -36,7 +36,9 @@ class LocalPubSubSpec extends ScalaTestWithActorTestKit with AnyWordSpecLike wit
         val statsProbe = testKit.createTestProbe[TopicImpl.TopicStats]()
         statsProbe.awaitAssert {
           fruitTopic ! TopicImpl.GetTopicStats(statsProbe.ref)
-          statsProbe.receiveMessage().localSubscriberCount should ===(3)
+          val stats = statsProbe.receiveMessage()
```

Comment:
how this is waiting?
the code is equivalent if I understand it correctly

Reply:
`awaitAssert` runs the block until it succeeds (or times out).

Previously it only waited for `localSubscriberCount should ===(3)`, now it also waits for `topicInstanceCount should ===(1)`

Reply:
I see!